ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -282,7 +282,7 @@ export default class ConfirmTransactionBase extends Component {
                 <TextField
                   type="number"
                   min="0"
-                  placeholder={ this.context.t('nonceFieldPlaceholder') }
+                  placeholder={ customNonceValue || nextNonce }
```

Comment:
Why is `customNonceValue` used as a placeholder? Presumably if `customNonceValue` is truthy, it'll be set as teh actual value of this field and the placeholder won't be shown at all.

Additionally, I think the `nextNonce` should be removed as a fallback `value` on line 296 if we want to fallback to the tx controller choosing the nonce as usual.

Reply:
fixed in f212ab8e9